virtool/api/response.py

```diff
@@ -22,6 +23,15 @@ def json_response(data: object, status: int = 200, headers: Optional[dict] = Non
     return resp
 
 
+class HTTPInsufficientRights(HTTPForbidden):
+    def __init__(self, message):
+        super().__init__(text=message, reason="insufficient_rights")
+
+
+def insufficient_rights(message="Insufficient rights"):
+    return HTTPInsufficientRights(message)
```

Comment:
Can we not just have:
```py
class HTTPInsufficientRights(HTTPForbidden):
    def __init__(self, message = "Insufficient rights"):
        super().__init__(text=message, reason="insufficient_rights")
```
Then raise with:
```py
raise HTTPInsufficientRights("Non-default message")
```

Is the helper function necessary?

Reply:
Hadn't thought of that, I'll give it a go.